kubeflow/trainer/config/local_runtimes/torch_distributed.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: trainer.kubeflow.org/v1alpha1
```

Comment:
Instead of installing the runtimes, can we just read the image version from GitHub dynamically ?

Reply:
Let me look into that. For offline support should we fall back to providing this?

Reply:
Is it okay to hardcode constants that will then need to be updated manually later? Something like this:
```
DEFAULT_FRAMEWORK_IMAGES = {
      "torch": "pytorch/pytorch:2.7.1-cuda12.8-cudnn9-runtime",
  }
```
This would then be used to create a default runtime, if the user doesn't provide a runtime url (and we have removed the sample one).

Reply:
I've pushed changes to use a default image if a user doesn't provide a URL and have removed the yaml. Please review. Thanks